en/assembly/quick_start_cuav_v5_nano.md

```diff
@@ -18,10 +18,10 @@ We'll go through each of these in detail in the following sections.
 | PM2 | [Do not use with PX4](../flight_controller/cuav_v5_nano.md#issue_pm2) |
 | TF CARD | SD card for log storage (comes with card) |
 | M1~M8 | PWM outputs. Can be used to control motors or servos. |
-| A1~A3 | PWM outputs. Can be used to control motors or servos. |
-| nARMED | |
+| A1~A3 | Capture pins (not supported on PX4). |
+| nARMED | Digital OUTPUT: GPIO will be set as input while not armed. While armed it will be configured as a GPIO OUT set low. |
```

Comment:
@hamishwillee I did not think you would copy it verbatim :)
Indicates the FMU is armed state. It is active low: low when armed.

Reply:
That's why I asked what it is :-)